components/webapp-mgt/org.wso2.carbon.webapp.list.ui/src/main/resources/web/webapp-list/webapp_info.jsp

```diff
@@ -104,6 +104,14 @@
             }
             else if (webappState.equalsIgnoreCase("started")) {
                 webapp = client.getStartedWebapp(webappFileName,hostName);
+                if (webappType.equalsIgnoreCase("JaxWebapp")) {
+                    webAppDataExtractor.getServletXML(client.getWarFileInputStream
+                            (webapp.getWebappFile(), hostName, webappType));
+                    wsdlURLS= webAppDataExtractor.getWSDLs(urlPrefix + webapp.getContext() + servletContext);
+                    wadlURLS= webAppDataExtractor.getWADLs(urlPrefix + webapp.getContext() + servletContext);

```

Comment:
Fixed with https://github.com/wso2/carbon-deployment/pull/216 and https://github.com/wso2/carbon-deployment/pull/217